doonut-core/src/main/java/com/doonutmate/doonut/member/repository/MemberRepository.java

```diff
@@ -14,4 +15,8 @@ public interface MemberRepository extends JpaRepository<MemberEntity, Long> {
                 WHERE member.deleted = false AND member.oauthInfo.oauthId = :oauthId
             """)
     Optional<MemberEntity> findByOauthId(String oauthId);
+
+    @Modifying
+    @Query("UPDATE MemberEntity m SET m.name = :newName WHERE m.id = :memberId")
+    void updateMemberNameByMemberId(Long memberId, String newName);
```

Comment:
사소한 부분이긴 한데 name하고 memberId 순서 바꾸는게 가독성이 더 좋을 것 같아요.  
```suggestion
    void updateMemberNameByMemberId(String newName, Long memberId);
```